modules/installation-mirror-repository.adoc

```diff
@@ -55,8 +56,35 @@ the pull secret for your mirror registry that you created.
 `ocp-release`.
 <7> For `server_architecture`, specify the architecture of the server, such as `x86_64`.
 
+. Mirror the version images to the internal container registry:
+** If your mirror host does not have internet access, take the following actions:
+... Connect the removable media to a system that is connected to the internet.
+... Review the images and configuration manifests to mirror:
++
+----
+$ oc adm -a ${LOCAL_SECRET_JSON} release mirror
+--from=quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE}
+--to=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}
+--to-release-image=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}:${OCP_RELEASE}-${ARCHITECTURE} --run-dry
+----
+... Record the entire `imageContentSources` section from the output of the previous
+command. The information about your mirrors is unique to your mirrored repository, and you must add the `imageContentSources` section to the `install-config.yaml` file during installation.
+... Mirror the images to a directory on the removable media:
++
+----
+$ oc adm -a ${LOCAL_SECRET_JSON} release mirror
```

Comment:
This command is for mirror-to-registry case, should be:
`
$ oc adm release mirror -a ${LOCAL_SECRET_JSON} --to-dir=${REMOVABLE_MEDIA_PATH}/mirror quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE}
`
right?